_posts/2022-11-01-ndk_advanced_custom_fields.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: "[CVE-2022-40839] Improper neutralization of SQL parameter in NdkAdvancedCustomizationFields module for PrestaShop"
+categories: module
+author:
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,ndk_advanced_custom_fields"
+severity: "critical (9.8)"
+---
+
+In NdkAdvancedCustomizationFields module for PrestaShop before 4.1.7, an anonymous user can perform a SQL injection in affected versions. 4.1.7 fixed vulnerabilities.
+
+## Summary
+
+* **CVE ID**: [CVE-2022-40839](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-40839)
+* **Published at**: 2022-11-01
+* **Advisory source**: [@daaaalllii](https://github.com/daaaalllii/cve-s/blob/main/CVE-2022-40839/poc.txt)
+* **Platform**: PrestaShop
+* **Product**: ndk_advanced_custom_fields
+* **Impacted release**: <= 4.1.6 (4.1.7 fixed vulnerability)
+* **Product author**: ndk design
+* **Weakness**: [CWE-89](https://cwe.mitre.org/data/definitions/89.html)
+* **Severity**: critical (9.8)
+
+## Description
+
+In NdkAdvancedCustomizationFields module for PrestaShop up to 4.1.6, a sensitive SQL call on `NdkCf` class can be executed with a trivial http call and exploited to forge a blind SQL injection throught for instance the POST or GET submitted `height` and `width` variables.
```

Comment:
In the NdkAdvancedCustomizationFields module for PrestaShop up to version 4.1.6, a sensitive SQL call in the `NdkCf` class can be executed via a trivial HTTP call. This vulnerability can be exploited to initiate a blind SQL injection, for instance, through the POST or GET submitted `height` and `width` variables.